src/modules/displaymanager/main.py

```diff
@@ -103,14 +117,23 @@ def set_autologin(username, displaymanagers, default_desktop_environment, root_m
                 for line in text:
                     if '[daemon]' in line:
                         if do_autologin:
-                            line = "[daemon]\nAutomaticLogin={!s}\nAutomaticLoginEnable=True\n".format(username)
+                            line = (
+                                "[daemon]\n",
+                                "AutomaticLogin={!s}\n",
```

Comment:
This replaces one string with a 3-tuple, and will break in the write() call later. You can use multiple string literals here.